src/main/java/org/owasp/esapi/logging/java/JavaLogFactory.java

```diff
@@ -93,43 +93,24 @@ public class JavaLogFactory implements LogFactory {
 
         LOG_BRIDGE = new JavaLogBridgeImpl(JAVA_LOG_APPENDER, JAVA_LOG_SCRUBBER, levelLookup);
 
-        readLoggerConfiguration(LogManager.getLogManager());
-    }
-
-    /**
-     * Attempts to load the expected property file path into the provided LogManager reference.
-     * @param logManager LogManager which is being configured.
-     */
-    /*package*/ static void readLoggerConfiguration(LogManager logManager) {
-        if (System.getProperties().keySet().stream().anyMatch(propKey ->
-        "java.util.logging.config.class".equals(propKey) || "java.util.logging.config.file".equals(propKey))) {
-            // LogManager has external configuration.  Do not load ESAPI defaults.
-            // See javadoc for the LogManager class for more information on properties.
-            boolean isStartupSysoutDisabled = Boolean.valueOf(System.getProperty(PropNames.DISCARD_LOGSPECIAL, Boolean.FALSE.toString()));
-            if (!isStartupSysoutDisabled) {
-                String logManagerPreferredMsg = String.format("[ESAPI-STARTUP] ESAPI JavaLogFactory Configuration will not be applied. "
-                        + "java.util.LogManager configuration Detected. "
-                        + "{\"java.util.logging.config.class\":\"%s\",\"java.util.logging.config.file\":\"%s\"}",
-                        System.getProperty("java.util.logging.config.class"), System.getProperty("java.util.logging.config.file"));
-
-                System.out.println(logManagerPreferredMsg);
-                // ::SAMPLE OUTPUT::
-                //[ESAPI-STARTUP] ESAPI JavaLogFactory Configuration will not be applied.  java.util.LogManager configuration Detected.{"java.util.logging.config.class":"some.defined.value","java.util.logging.config.file":"null"}
-            }
-
-            return;
-        }
         /*
-         * This will load the logging properties file to control the format of the output for Java logs.
+         * esapi-java-logging.properties file may lead to confusing logging behavior
+         * by overriding desired configurations provided through Java's LogManager class.
+         * 
+         * Verify the file is not present and fail if found to enforce understanding of
+         * the configuration method.
          */
         try (InputStream stream = JavaLogFactory.class.getClassLoader().
                 getResourceAsStream("esapi-java-logging.properties")) {
-            if (stream == null) {
-                throw new ConfigurationException("Unable to locate resource: esapi-java-logging.properties");
+            if (stream != null) {
+                throw new ConfigurationException("esapi-java-logging.properties is no longer supported.  See https://github.com/ESAPI/esapi-java-legacy/wiki/Configuring-the-JavaLogFactory for information on corrective actions.");
```

Comment:
I think it would make more sense to define this exception message as `final String ` and the reference it both places. That way, if we every decide to change the error message, we won't forget to change one of the instances. Otherwise, I think this is great and hopefully it will head off questions from those who seem to have a habit of never reading the release notes.

Reply:
Update applied